frontend/src/app/Settings/types.d.ts

```diff
@@ -150,3 +150,8 @@ type OrganizationType =
   | "camp"
   | "lab"
   | "other";
+
```

Comment:
I didn't know that the d.ts use was discouraged in typescript because the context of the type can be messed up. Sounds like these files are more like a result of typescript being compile into JS.

Reply:
Yeah so based on the reading I did, the `.d.ts` files are there for libraries where typescript and javascript files are sitting side by side. In cases where JS files are necessary, adding the `.d.ts` files allows you to still have access to the relevant types while not requiring those files to be typescript files. 

Based on what I found this pattern isn't as big an issue for applications since the main antipattern is global access when libraries are imported (which would make those variables available at the global namespace even outside the library) but I think it's still worth it to make it explicit where these new types are coming from just so there isn't any confusion.